filters/apiusagemonitoring/noop.go

```diff
@@ -0,0 +1,20 @@
+package apiusagemonitoring
+
+import "github.com/zalando/skipper/filters"
+
+type noopSpec struct{
+	filter filters.Filter
+}
+
+func (*noopSpec) Name() string {
+	return Name
+}
+
+func (s *noopSpec) CreateFilter(config []interface{}) (filters.Filter, error) {
+	return s.filter, nil
+}
+
+type noopFilter struct{}
+
+func (*noopFilter) Request(filters.FilterContext)  {}
```

Comment:
i find the pointer here unnecessary, but it's fine

Reply:
Just made it a "rule" to have receivers by pointers in general to avoid structure content copy. In this case though, quite sure the compiler would not pass it at all since there is ⓵ no value at all in the struct ⓶ no access to it at all anyways. So guess both would be OK.